bedrock/firefox/views.py

```diff
@@ -372,6 +372,9 @@ def get_context_data(self, **kwargs):
         # add version to context for use in templates
         ctx['version'] = self.kwargs.get('version') or ''
 
+        # add funnelcake to context for use in templates
+        ctx['funnelcake'] = self.request.GET.get('f', '')
```

Comment:
Yup. We have [a context processor](https://github.com/mozilla/bedrock/blob/master/bedrock/mozorg/context_processors.py#L33-L46) that automatically puts `funnelcake_id` in the template context when `f=123` is in the URL query params. This should be removed and `funnelcake_id` used in the template.